connect-rest-extension/README.md

```diff
@@ -0,0 +1,39 @@
+[![License](http://img.shields.io/:license-apache%202.0-brightgreen.svg)](http://www.apache.org/licenses/LICENSE-2.0.html)
+[![Build Status](https://travis-ci.com/debezium/debezium-kcd-rest-extension.svg?branch=master)](https://travis-ci.com/debezium/debezium-kcd-rest-extension/)
```

Comment:
We don't use Travis any longer. In fact, I'd suggest to remove all these links, we have them in the root README, and there's no value in duplicating the information.

Reply:
copy & paste issue